python/minpy/__init__.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
```

Comment:
@hotpxl @zzhang-cn
We agree that the minpy "folder" will look like this:
```
    minpy
         |- autograd.py
         |- other stuffs in package root ....
         |- nn
           |- model/solver/optim/ ...
         |- jit
           |- jit related stuffs ...
```
I think the only question is whether it is `minpy` or `mxnet.minpy`. For simplicity, `minpy` is shorter. And since `minpy` is quite standalone (provides autograd/jit/pytorch-like interface), so it is fair to have a standalone namespace. However, Eric mentioned that this will result in installing two packages. I am not sure about this.

If we look at tensorflow and tensorboard, they have totally separate folder. I guess the reason is because not all the people want to install tensorboard. Then the question is will we have similar situation? (some people just need symbols so minpy can be optionally turned off).

Want to hear your suggestions.